NEWS.md

```diff
@@ -9,6 +9,10 @@
   server requirements where Snowflake tokens provide proxied authentication
   while API keys identify users to the Connect server itself.
 
+* Fix an opaque error when creating a manifest in a project using Python <= 3.10 with
```

Comment:
NOTE: This NEWS update landed in the wrong location since a patch release occurred while this PR was in-flight. Adjusting as we go to release 1.6.2.